Mods/src/PuppiMod.cc

```diff
@@ -186,7 +190,35 @@ PuppiMod::Process()
 
   auto* pv = vertexes->At(0);
 
-  auto* pfCandidates = GetObject<PFCandidateCol>(fPFCandidatesName);
+  PFCandidateArr *InvertedParticles = new PFCandidateArr;
```

Comment:
This needs to move to SlaveBegin (so InvertedParticles must become a class member). Should treat it the same way as you do for fPuppiParticles.